src/number-formatting/format.js

```diff
@@ -12,6 +12,37 @@ function formatNegativeNumber(str, options) {
 }
 
 /**
+ * converts a number to a decimal string if it is on scientific notation
+ * (method taken from https://gist.github.com/jiggzson/b5f489af9ad931e3d186)
+ * @param number
+ */
+function convertScientificToDecimal(number) {
+    // if the number is in scientific notation remove it
+    if(/\d+\.?\d*e[\+\-]*\d+/i.test(number)) {
```

Comment:
is number coerced to a string? I'd prefer forcing it to a string at the top so it is clear (see other comment)